client/app/components/blueprints/blueprint-editor/_blueprint-editor.sass

```diff
@@ -1,3 +1,6 @@
+.capitalize
```

Comment:
I'm looking for camel case/initial caps.  Couldn't find an angular filter for that :frowning: